packages/manager/src/features/CloudPulse/shared/CloudPulseCustomSelect.tsx

```diff
@@ -237,6 +237,9 @@ export const CloudPulseCustomSelect = React.memo(
             placement: 'bottom',
           },
         }}
+        textFieldProps={{
+          required: true,
+        }}
```

Comment:
```suggestion
        required
```

no need to go inside textFieldProps, just directly use  `required` props. Change it all the places